pycytominer/tests/test_data/cytominer_database_example_data/shrink_SQ00014613.sqlite_for_tests.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
I want to confirm that this function isn't executed during testing. Is that correct?

Reply:
Correct, this file doesn't get executed during testing due to not having a matching [Pytest default filename pattern](https://docs.pytest.org/en/7.1.x/getting-started.html#run-multiple-tests). This may be validated through the command `pytest --collect-only`. To be extra careful I'll remove `tests` from the filename to make this more explicit. We could get more specific here with a [Pytest configuration file](https://pytest.org/en/latest/reference/customize.html) as well, though this may be better suited for an external issue's focus.

Reply:
Just a note here: filename was changed to avoid any issues with pytest runs.